src/redux/middlewares.js

```diff
@@ -107,3 +107,60 @@ export const scrollMiddleware = store => next => action => {
   }
   return next(action)
 }
+
+import {init} from '../services/realtime'
+
+const bindHandlers = dispatch => ({
+  'post:new': data => dispatch({type: ActionTypes.REALTIME_POST_NEW, post : data.posts}),
+  'post:update': data => dispatch({type: ActionTypes.REALTIME_POST_UPDATE, post : data.posts}),
+  'post:destroy': data => dispatch({type: ActionTypes.REALTIME_POST_DESTROY, postId: data.meta.postId}),
+  'post:hide': data => dispatch({type: ActionTypes.REALTIME_POST_HIDE, postId: data.meta.postId}),
+  'post:unhide': data => dispatch({type: ActionTypes.REALTIME_POST_UNHIDE, postId: data.meta.postId}),
+  'comment:new': data => dispatch({type: ActionTypes.REALTIME_COMMENT_NEW, comment: data.comments}),
+  'comment:update': data => dispatch({type: ActionTypes.REALTIME_COMMENT_UPDATE, comment: data.comments}),
+  'comment:destroy': data => dispatch({type: ActionTypes.REALTIME_COMMENT_DESTROY, commentId: data.commentId}),
+  'like:new': data => dispatch({type: ActionTypes.REALTIME_LIKE_NEW, postId: data.meta.postId, user: data.users}),
+  'like:remove': data => dispatch({type: ActionTypes.REALTIME_LIKE_REMOVE, postId: data.meta.postId, userId: data.meta.userId}),
+})
+
+export const realtimeMiddleware = store => {
+  let realtimeConnection
```

Comment:
Well, I'd say _here_ is fine to use `socketConnection`, since it is. But you would probably think I'm messing with your head, so let's leave it as it is. :)